guides/common/modules/ref_overview-of-the-host-columns.adoc

```diff
@@ -0,0 +1,46 @@
+[id="overview-of-the-host-columns_{context}"]
+= Overview of the Host Columns
+
+Below is the complete overview of columns that can be displayed in the host table divided into content categories.
```

Comment:
Should this be sorted in alphabetical order?

Maybe also add "For more information on adding columns to the host overview page, see xref:...[]."

Reply:
I can add the xref, good idea. As far as sorting goes, I used the exact order of the Project web UI. It could also be sorted alphabetically but I think this way user can find the description faster if they look for a specific column.

Reply:
I am fine either way, as long as it's not random. If you go with the order like in the Web UI, please add a small comment to the source file.

Reply:
Added :+1: